images/hail-gcp-notebook/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM gcr.io/deeplearning-platform-release/base-cpu
+
+RUN apt-get update && \
+    apt-get install -y openjdk-8-jre-headless g++ python3-pip libopenblas-base liblapack3 && \
+    rm -rf /var/lib/apt/lists/* && \
+    pip3 install "cloudpathlib[all]" hail phantomjs pyarrow && \
+    cd $(find_spark_home.py)/jars && \
+    curl -O https://storage.googleapis.com/hadoop-lib/gcs/gcs-connector-hadoop2-2.0.1.jar
```

Hadolint is wanting:

```suggestion
    curl -o "$(find_spark_home.py)/jars/gcs-connector-hadoop2-2.0.1.jar" https://storage.googleapis.com/hadoop-lib/gcs/gcs-connector-hadoop2-2.0.1.jar
```

---

That's a surprisingly smart linter suggestion :)